css-typed-om/Overview.bs

```diff
@@ -899,7 +899,7 @@ The following are the arithmetic operations you can perform on dimensions:
         whose {{CSSUnitValue/unit}} internal slot is set to |this|’s {{CSSUnitValue/unit}} internal slot,
         and {{CSSUnitValue/value}} internal slot is set to the sum of
         the {{CSSUnitValue/value}} internal slots
-        of the [=list/items=] in |values|.
+        of the [=list/items=] in |values| (from left to right).
```

Comment:
Does "from left to right" mean "a + (b + c)" or "(a + b) + c"?  It would be better to just make this explicit instead of trying to be terse.